CompetitionTemplate/src/competition/subsystems/arm/arm_commands/DropKickstandCommand.java

```diff
@@ -0,0 +1,67 @@
+package competition.subsystems.arm.arm_commands;
+
+import com.google.inject.Inject;
+
+import competition.subsystems.arm.ArmSubsystem;
+import edu.wpi.first.wpilibj.Timer;
+import xbot.common.command.BaseCommand;
+import xbot.common.properties.DoubleProperty;
+import xbot.common.properties.XPropertyManager;
+
+public class DropKickstandCommand extends BaseCommand {
+
+    ArmSubsystem arm;
+    
+    DoubleProperty kickstandDropHeight;
+    DoubleProperty kickstandRaisePower;
+    DoubleProperty kickstandRaiseDuration;
+    
+    double startTime;
+    
+    @Inject
+    public DropKickstandCommand(ArmSubsystem arm, XPropertyManager propMan) {
+        kickstandDropHeight = propMan.createPersistentProperty("KickstandDropHeight", 5.0);
```

Comment:
raiseAngle or something? I feel like it should be more clear this is a delta vs an absolute height